Codeinwp/neve
# IE primary nav alignment | AMP Escaping issues

### Summary
Please check the commits below.


### Will affect visual aspect of the product

NO

### Test instructions


- Test AMP
- Test on IE - for menu centering
- Try importing Neve Original - with the new version of the Onboarding (1.3.5) which should be fetched on `composer install`. Also please check the logs under `/uplods/ti_theme_onboarding.log` 💃 


Closes #503, #501.
